README.md

```diff
@@ -46,6 +46,21 @@ Any rpm package placed into the additional-packages/ directory will be uploaded
 
 **NOTE**: All packages must end with extension `"$(uname -m).rpm"`, ie `.x86_64.rpm` or `.aarch64.rpm`.
 
+## ECS Logs Collector
+
+The ECS logs collector is a shell script that helps gather diagnostic information for troubleshooting ECS-related issues. This script is automatically installed on all ECS-optimized AMIs built with this recipe.
+
+### Installation Details
+
+The ECS logs collector is installed during the AMI build process with the following characteristics:
+
+- **Installation Location**: `/opt/amazon/ecs/ecs-logs-collector.sh`
+- **Version Tracking**: A version file is created at `/opt/amazon/ecs/ECS_LOG_COLLECTOR_VERSION` containing the git commit hash (created during AMI build)
```

Comment:
`ECS_LOG_COLLECTOR_VERSION` - curious how this is being created and tracked

Reply:
https://github.com/aws/amazon-ecs-logs-collector

I don't believe this repo had a version and versioning is being introduced here for the first time, is that right?

non blocking nit: Another way of including version could be, we could add `version` as part of the script itself. and the version could be fetch-able as a command line argument `sh ecs-logs-collector.sh -v`

Reply:
Synced offline and as i understand, updating `ECS_LOG_COLLECTOR_VERSION` will be done as followup in syncing to codecommit